core/tsconfig.json

```diff
@@ -8,6 +8,7 @@
     "outDir": "dist",
     // MikroORM
     "experimentalDecorators": true,
+    "skipLibCheck": true,
```

Comment:
👍 

Would you be able to remove the `--skipLibCheck` flag in `build.yml` and `Makefile` then?

Reply:
I don't see the flag in those files. Can you check again?

Reply:
Sorry, I added them in the `main` branch recently, so you'd need to update this branch to see them

https://github.com/BemiHQ/bemi/blob/c0d6e7d78fef79192c2add3807722b743ba4f72e/Makefile#L4

Reply:
Yes, my bad that my branch was based on my fork reap. It's updated now.

Reply:
Sweet, thank you!